src/utils/chartOptions.js

```diff
@@ -171,47 +164,132 @@ export const barGraphOptions = options => merge({
  * Options ONLY for Doughnut chart
  * @param {object} options - More options that can be pass to the chart
  */
-export const doughnutGraphOptions = options => merge({
+export const doughnutChartOptions = options => merge({
   cutoutPercentage: 60,
 
   elements: {
     arc: {
-      backgroundColor: styles.ultramarineBlue,
+      backgroundColor: chartStyles.ultramarineBlue,
       borderAlign: 'center',
-      borderColor: styles.whiteColor,
+      borderColor: chartStyles.whiteColor,
       borderWidth: 1,
     },
   },
 }, baseOptions, options);
 
 
+/**
+ * data ONLY for Line chart
+ * @param {object} data - More data that can be pass to the chart
+ */
+export const lineChartData = data => merge({
+  datasets: [
+    {
+      backgroundColor: chartStyles.transparent,
+      borderColor: chartStyles.borderColor,
+      pointBorderColor: chartStyles.borderColor,
+      pointBackgroundColor: chartStyles.whiteColor,
+      pointHoverBackgroundColor: chartStyles.whiteColor,
+      pointHoverBorderColor: chartStyles.ultramarineBlue,
+      pointHoverBorderWidth: 4,
+      borderWidth: 2,
+    },
+  ],
+}, data);
+
+
+/**
+ * data ONLY for Bar chart
+ * @param {object} data - More data that can be pass to the chart
+ */
+export const barChartData = data => merge({
+  datasets: [
+    {
+      barPercentage: 1,
+      categoryPercentage: 0.5,
+      barThickness: 'flex',
+      maxBarThickness: 14,
+      minBarLength: 0,
+    },
+    {
+      barPercentage: 1,
+      categoryPercentage: 0.5,
+      barThickness: 'flex',
+      maxBarThickness: 14,
+      minBarLength: 0,
```

Comment:
🎨  I think this shouldn't be 0, according to the design https://projects.invisionapp.com/d/main?origin=v7#/console/18539694/385859990/inspect?scrollOffset=1048